onnxruntime/core/framework/session_state_utils.cc

```diff
@@ -89,6 +139,16 @@ static common::Status DeserializeTensorProto(const Env& env, const std::basic_st
 
   if (strcmp(p_tensor->Location().name, CPU) == 0) {
     // deserialize directly to CPU tensor
+    if (utils::HasExternalData(tensor_proto)) {
+      OrtCallback ext_data_deleter;
+      Status st = ExtDataTensorProtoToTensor(env, proto_path, tensor_proto, *p_tensor, ext_data_deleter);
```

Comment:
ORT_RETURN_IF_ERROR ?

Reply:
changed to `ORT_RETURN_IF_ERROR(ExtDataTensorProtoToTensor(...))`